apisix/api_router.lua

```diff
@@ -90,14 +90,18 @@ function fetch_api_router()
                            core.json.delay_encode(api_routes, true))
             for _, route in ipairs(api_routes) do
                 local typ_uri = type(route.uri)
-                if typ_uri == "string" then
-                    has_route_not_under_apisix =
-                        not core.string.has_prefix(route.uri, "/apisix/")
-                else
-                    for _, uri in ipairs(route.uri) do
+                if not has_route_not_under_apisix then
```

Comment:
IMHO, break if has_route_not_under_apisix is better?

Reply:
Maybe not，there is a lot of code under has_route_not_under_apisix value finder.
If we break after outer for loop，the code left will be skipped